AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenContext.m

```diff
@@ -20,7 +21,7 @@ @interface MSAuthTokenContext ()
 /**
  * The last value of user account id.
  */
-@property(nullable, nonatomic, copy) NSString *homeAccountId;
+@property(nullable, atomic, copy) NSString *homeAccountId;
```

Comment:
You moved this to header, shouldn't it be removed?

Reply:
No, it's readonly in header

Reply:
Ah.. OK.

Reply:
The case is the default `readwrite` attribute outside of the header. The setter should stay private